packages/react/src/Token/Token.tsx

```diff
@@ -96,8 +96,11 @@ const Token = forwardRef((props, forwardedRef) => {
           <LeadingVisual />
         </LeadingVisualContainer>
       ) : null}
-      <TokenTextContainer {...(hasMultipleActionTargets ? interactiveTokenProps : {})}>{text}</TokenTextContainer>
-      {onRemove && <VisuallyHidden> (press backspace or delete to remove)</VisuallyHidden>}
+      <TokenTextContainer {...(hasMultipleActionTargets ? interactiveTokenProps : {})}>
+        {text}
+        {onRemove && <VisuallyHidden> (press backspace or delete to remove)</VisuallyHidden>}
+      </TokenTextContainer>
```

Comment:
Just confirming this change: What does bringing onRemove inside the TokenTextContainer give us?

Reply:
Ah, understood.

Let's run integration once just to make sure there are no jest tests that expect just the text to be present in the token